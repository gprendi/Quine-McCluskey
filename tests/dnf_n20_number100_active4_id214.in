d 20
00010000000000000000
00010000010100000000
01000000000100000000
00000000010100000000
01000000010000000000
00000000000000000000
00010000010000000000
01010000000000000000
00010000000100000000
00000000000100000000
01000000010100000000
01000000000000000000
01010000010000000000
01010000010100000000
01010000000100000000
00000000010000000000
