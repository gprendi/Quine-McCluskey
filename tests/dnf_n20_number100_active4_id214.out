p dnf 20 0
-1 -3 -5 -6 -7 -8 -9 -11 -13 -14 -15 -16 -17 -18 -19 -20 0
